scripts/portconfig

```diff
@@ -213,6 +220,26 @@ class portconfig(object):
         state_db.connect(state_db.STATE_DB)
         return state_db.get(state_db.STATE_DB, '{}|{}'.format(PORT_STATE_TABLE_NAME, port), PORT_STATE_SUPPORTED_SPEEDS)
 
+    def get_supported_fecs(self, port):
+        # If there is supported_fecs exposed in STATE_DB, let's use it.
+        # Otherwise, take the default
+        if not self.namespace:
+            state_db = SonicV2Connector(host="127.0.0.1")
+        else:
+            state_db = SonicV2Connector(host="127.0.0.1", namespace=self.namespace, use_unix_socket_path=True)
+        state_db.connect(state_db.STATE_DB)
+
+        supported_fecs_str = state_db.get(state_db.STATE_DB, '{}|{}'.format(PORT_STATE_TABLE_NAME, port), PORT_STATE_SUPPORTED_FECS)
+        if supported_fecs_str:
```

Comment:
After your swss change, the support_fecs_str will be either the list of fec or 'N/A' if SAI is not supporting the query. In case when SAI is not supporting it, an empty list is returned here. Instead we should be returning the list with rs,fc and none. This is needed for backward compatibility